zeroex/order_test.go

```diff
@@ -11,54 +13,57 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGenerateOrderHash(t *testing.T) {
-	fakeExchangeContractAddress := common.HexToAddress("0x1dc4c1cefef38a777b15aa20260a54e584b16c48")
+var fakeExchangeContractAddress = common.HexToAddress("0x1dc4c1cefef38a777b15aa20260a54e584b16c48")
 
-	order := Order{
-		MakerAddress:          constants.NullAddress,
-		TakerAddress:          constants.NullAddress,
-		SenderAddress:         constants.NullAddress,
-		FeeRecipientAddress:   constants.NullAddress,
-		MakerAssetData:        constants.NullAddress.Bytes(),
-		TakerAssetData:        constants.NullAddress.Bytes(),
-		ExchangeAddress:       fakeExchangeContractAddress,
-		Salt:                  big.NewInt(0),
-		MakerFee:              big.NewInt(0),
-		TakerFee:              big.NewInt(0),
-		MakerAssetAmount:      big.NewInt(0),
-		TakerAssetAmount:      big.NewInt(0),
-		ExpirationTimeSeconds: big.NewInt(0),
-	}
+var testOrder = &Order{
+	MakerAddress:          constants.GanacheAccount0,
+	TakerAddress:          constants.NullAddress,
+	SenderAddress:         constants.NullAddress,
+	FeeRecipientAddress:   constants.NullAddress,
+	MakerAssetData:        constants.NullAddress.Bytes(),
+	TakerAssetData:        constants.NullAddress.Bytes(),
+	ExchangeAddress:       fakeExchangeContractAddress,
+	Salt:                  big.NewInt(200),
```

Comment:
I changed these numbers just in case. It's hypothetically easier to get a false positive when testing using zeroes for all the numbers.